mappings/net/minecraft/server/world/PlayerChunkWatchingManager.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS uh net/minecraft/server/world/PlayerChunkWatchingManager
```

Comment:
Is `tracking` is used for tracking entities? It's probably beneficial to use consistent terminology with that.

Reply:
I'd say tracking makes more sense than watching here, but we can change it in Entity too.

Reply:
The current idea is that a player can watch a chunk and will receive updates about it. We can change that to tracking if that makes more sense. Alternatively, maybe we can borrow "observer" from the observer pattern?

Reply:
Could be completely wrong, but I seem to vaguely recall tracking being specifically mentioned in some logging. Seems to be the most widely accepted description -- I imagine it's worth using.